### ISSUE TITLE
Villagers will not pick up crops when Liteloader and Harvestcraft are both installed on 1.12.2.

### ISSUE THREAD

**Issue by xkforce** [MatrexsVigil/harvestcraft#85]
Minecraft 1.12.2

Harvestcraft 1.12.2a

LWJGL 2.9.1

Forge 1.12.2-14.23.1.2555

Liteloader (snapshot) which is the only version available for 1.12.2

Bug report by different user for older version with the same issue located at: https://github.com/MatrexsVigil/harvestcraft/issues/44 which is marked closed.

How the issue was reproduced:

* clean install of 1.12.2

* installed latest version of Harvestcraft with no other mods.

* Villagers will pick up crops with harvestcraft installed but not liteloader.

* Villagers will pick up crops with liteloader installed but not harvestcraft.

* Villagers will not pick up crops with both of them installed at the same time.

Liteloader was installed because some useful hud mods required it:

https://minecraft.curseforge.com/search?search=liteloader

**Comment by tabbykitty1** [comment]
Yes, it is a problem with LiteLoader, there's a couple other problems it causes too. Various crops don't drop anything but seeds being one of them when right-clicking, and unable to harvest trees because they require right-clicking to harvest.

**Comment by ghost** [comment]
In Theory, mods for Lite Loader only can be made for Forge by changing the extension (.litemod to .jar)

EDIT: 
As I said, In theory. I tried to do it but it didn't work, although Forge loaded correctlly it didn't get the mod into the Mod directory.
